NuclearData/xsPackages/IMCXsPackages_class.f90

```diff
@@ -0,0 +1,255 @@
+!!
+!! This module brakes standard rules
+!! It contains a library of XS Packages for IMC particle type
+!!
+!! Pretty much a copy of neutronXsPackages_class, may contain unnecessary lines
+!!
+module IMCXsPackages_class
+
+  use numPrecision
+  use endfConstants
+
+  implicit none
+  private
+
+  !!
+  !! IMC MACROscopic Reaction XSS
+  !!
+  !! Public Members:
+  !!   total            -> total Cross-Section [1/cm]
```

Comment:
You can remove elastic and inelastic and add Planck opacity. Perhaps you could also add the Planck function too for easier sampling?